lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -276,6 +276,10 @@ export abstract class ClientApplication {
                     this.nativeExtensionProvider = undefined; // If extension gets uninstalled during session prevent future requests from continuing to attempt
                     const redirectClient = this.createRedirectClient(request.correlationId);
                     return redirectClient.acquireToken(request);
+                } else if (e instanceof InteractionRequiredAuthError) {
+                    this.logger.verbose("acquireTokenRedirect - Resolving interaction required error thrown by native broker by falling back to web flow");
+                    const popupClient = this.createRedirectClient(request.correlationId);
```

Comment:
typo: redirecClient??

Reply:
Oops...lazy copy pasting. Thanks for catching this!